external_trigger: false

#joint_estimator: FeatureBasedJointEstimator
joint_estimator: RecursiveJointEstimator
joint_estimator_in_re: TransformationBasedJointEstimator
#joint_estimator_in_re: FeatureBasedJointEstimator


# Interval selection:
# ONA (Overlapping & NonAdvancing) = 0
# NOA (NonOverlapping & Advancing) = 1
# OA (Overlapping & Advancing) = 2
interval_selection: 0
interval_selection_overlap: 10
